scripts/fires/wfigs_data.py

```diff
@@ -101,33 +129,44 @@ def get_place(x):
             if not (pd.isna(x.InitialLatitude) or pd.isna(x.InitialLongitude)
                    ) and abs(x.InitialLatitude) <= 90 and abs(
                        x.InitialLongitude) <= 180:
+                if str(x.InitialLatitude) in _CACHE:
```

Comment:
please use store the output of `str(x.InitialLatitude)` and use that rather calling this over and over..

Reply:
done